lib/rails_api_auth.rb

```diff
@@ -41,4 +41,9 @@ module RailsApiAuth
   # currently only accepts :uuid
   mattr_accessor :primary_key_type
 
+  # @!attribute [rw] force_ssl
+  # Force SSL for Oauth2Controller.
+  mattr_accessor :force_ssl
```

Comment:
Not seeing how to "officially" mark something as default - http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md I can just make a note, I guess... So something like

`# Force SSL for Oauth2Controller. Default: true`

Is that what you had  in mind?